.circleci/config.yml

```diff
@@ -158,7 +158,11 @@ jobs:
             export GOOGLE_APPLICATION_CREDENTIALS=$HOME/extra/appcredentials.json
             hack/go-generate.sh
             go mod download
-            gotestsum --junitfile /tmp/test-results/go-test-report.xml --format short-with-failures -- -race ./...
+            # cafs tests are memory intensive: best to run them separately
+            gotestsum --junitfile /tmp/test-results/go-test-report.xml --format short-with-failures -- \
+              -race $(go list ./...|grep -v cafs)
```

Comment:
nit:  number of source files now limited by shell's maximum number of parameters

Reply:
counter-nit: `go list` yields the number of packages

Reply:
follow-up note: comment only. No change requested